docs/tutorial/django.rst

```diff
@@ -39,6 +39,10 @@ install ``python3-venv`` and create a virtual environment:
     :end-before: [docs:create-venv-end]
     :dedent: 2
 
+.. note::
+   A pop-up window may appear when installing ``python-venv``. Select ``OK``
```

Comment:
this is news to me: what is this pop-up window?

Reply:
Based on [the code it's running](https://github.com/canonical/rockcraft/blob/e75db7fd3fb5f7ff26ad1eb9c4f02d51208c81a9/docs/tutorial/code/django/task.yaml#L13-L18) I would presume it's an account authorisation window for `sudo`, but AFAIK we don't have that by default on any Ubuntu flavours. Maybe this is changing in oracular? (is it time for me to switch to oracular?)

Reply:
Perhaps @erinecon you could post a screenshot?

Reply:
![Screenshot from 2024-09-23 09-01-25](https://github.com/user-attachments/assets/96a8c5ca-6106-42b3-b542-753092c422de)

Reply:
Yes this is a fresh Multipass VM. When you say to start the tutorial with updating the VM, do you mean something other than `sudo apt-get update`?

Reply:
I did a fresh Multipass VM, followed instructions up to `sudo apt-get update` and then also did `sudo apt-get upgrade -y`. This is the pop-up window that occurs from that command:
![Screenshot from 2024-09-23 10-10-12](https://github.com/user-attachments/assets/1ef6ec12-c165-4f47-acc8-913a0787c740)

Then, when I do `sudo apt-get install python3-venv -y`, I still get a pop-up window, but it's different now:
![Screenshot from 2024-09-23 10-12-01](https://github.com/user-attachments/assets/a7ed9d68-b4d6-4eb8-8f23-33b4a7ea74f2)

Reply:
We discussed this with the Charmcraft team and they suggested not to mention this and let people figure it out on their own

Reply:
Sounds good, I removed the note